submission/submission-security/src/main/kotlin/ebi/ac/uk/security/integration/SecurityModuleConfig.kt

```diff
@@ -42,12 +44,15 @@ class SecurityModuleConfig(
     val userRegister: Observable<UserRegister> = Events.userPreRegister
 
     private val groupService by lazy { GroupService(groupRepository, userRepo) }
-    private val securityService by lazy { SecurityService(userRepo, securityUtil, props, profileService) }
+    private val securityService by lazy {
+        SecurityService(userRepo, securityUtil, props, profileService, captchaVerifier)
+    }
     private val securityFilter by lazy { SecurityFilter(props.environment, securityService) }
     private val userPrivilegesService by lazy {
         UserPrivilegesService(userRepo, tagsDataRepository, queryService, userPermissionsService)
     }
 
+    private val captchaVerifier by lazy { CaptchaVerifier(RestTemplate(), props) }
```

Comment:
Should we inject the `RestTemplate` instead of creating it directly? Like in the case of the `objectMapper`